karmada/kcl.mod

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "karmada"
+edition = "0.1.0"
+version = "0.1.0"
+
+[dependencies]
+k8s = "1.27"
```

Comment:
U can use `kcl mod add k8s:1.27` with the kcl v0.7.0 version to update the kcl.mod.lock file.